src/agent/Agent.ts

```diff
@@ -156,6 +159,10 @@ export class Agent {
     await this.wallet.delete()
   }
 
+  public closeSession(session: TransportSession) {
```

Comment:
`closeSession` is misleading as it won't close the session. `removeSession` on transport service makes more sense.

Also, I don't think we should expose the `removeSession` on the agent. It doesn't feel like public API to me, so would rather see the classes that need it inject it.

Reply:
Agree. Do you have any particular idea, could you give me an example of that "classes that need it inject it."?